uint8 forward_brightness # 0-255
uint8 internal_brightness # 0-127
uint8 mood_color # 0=off,1=green,2=yellow,3=red
